tests/parametric/test_headers_baggage.py

```diff
@@ -56,6 +57,26 @@ def test_headers_baggage_only_D002(self, test_library):
         assert "baggage" in headers.keys()
         assert headers["baggage"] == "foo=bar"
 
+    @irrelevant(
+        context.library == "cpp",
+        reason="Cpp does not support baggage at this time; therefore, we cannot disable baggage.",
```

Comment:
Having a more precise comment will help understand why we use irrelevant:

```suggestion
        reason="Cpp current default behaviour matches the future baggage disabled behaviour, so we can't activate this test without causing a false easy win",
```

You can even do it in a single line with `context.library in ("cpp", "golang", ...)` to prevent big boilerplate